src/screens/Topics/index.js

```diff
@@ -224,26 +222,14 @@ const styles = StyleSheet.create({
     paddingHorizontal: dimen.normalizeDimen(20),
     paddingBottom: dimen.normalizeDimen(20),
     backgroundColor: COLORS.white,
-    shadowColor: COLORS.black,
-    shadowOffset: {
-      width: 0,
-      height: 5
-    },
-    shadowOpacity: 0.36,
-    shadowRadius: 6.68,
-
-    elevation: 11,
     flexDirection: 'column',
     justifyContent: 'space-between'
   },
   textGetPersonalContent: {
-    fontFamily: 'Inter',
-    fontStyle: 'normal',
-    fontWeight: '400',
+    fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(24),
-    color: COLORS.blackgrey,
-    opacity: 0.84,
+    color: COLORS.gray500,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to remove the shadow effect from the component. If this was intentional and aligns with the design requirements, then it's fine. However, if the shadow was removed unintentionally, you might want to add it back.
```diff
+    shadowColor: COLORS.black,
+    shadowOffset: {
+      width: 0,
+      height: 5
+    },
+    shadowOpacity: 0.36,
+    shadowRadius: 6.68,
+    elevation: 11,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->